README.md

```diff
@@ -1 +1,8 @@
-# Hello-Microverse
\ No newline at end of file
+# :open_book: Hello-Microverse-Project
+Hello-Microverse project is a practical learning simulation covering the following in step by step commits:
+*GitHub flow
+*linter for HTML and CSS
+*.gitignore files
+
+:man_student: Author:
+*GitHub: @Thoth1111
```

Comment:
- [ ] Kindly please use [this Microverse template](https://github.com/microverseinc/readme-template) for your Readme file. This will give more details about your project..

Reply:
Hi @OyePriscilla,
Thank you for the great feedback. I have made the changes you've highlighted. I have not filled out all the details in the Readme file since some do not apply to the project, but I have kept all required sections in the file. I hope this is okay.
Thanks again.